comp/livestream/plan-video-feeds.yaml

```diff
@@ -0,0 +1,14 @@
+summary: Plan video feeds
+
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Competition Prep
+
+description: >-
+  The livestream will need to have a few different camera feeds to show different parts of the venue
+
+  - Arena (top down ideally, maybe more)
```

Comment:
This sounds like we should opt for top-down if there's only one, which I'm not sure is ideal? Suggest just leaving this without details.